src/main/java/com/conveyal/datatools/manager/controllers/api/UserController.java

```diff
@@ -32,7 +33,6 @@
 import org.slf4j.LoggerFactory;
 import spark.Request;
 import spark.Response;
-
```

Comment:
I think we typically include a new line between imports from dependencies and native java packages.

Reply:
Updated.